source/diagnostics.tex

```diff
@@ -1192,7 +1201,6 @@
 
   // \ref{syserr.errcode.nonmembers} non-member functions:
   error_code make_error_code(errc e) noexcept;
-  bool operator<(const error_code& lhs, const error_code& rhs) noexcept;
 
   template <class charT, class traits>
     basic_ostream<charT, traits>&
```

Comment:
Sorry, I wasn't very clear. The "this one" I'm referring to is the `operator<<` whose first line is highlighted here (github doesn't allow adding comments outside the diff context, annoyingly). I think that should be deleted, because you added it to the synopsis, but it hasn't been.